src/include/mallocMC/mallocMC_utils.hpp

```diff
@@ -186,4 +186,39 @@ namespace mallocMC
   template<class T>
   MAMC_HOST MAMC_ACCELERATOR inline T divup(T a, T b) { return (a + b - 1)/b; }
 
+  /** the maximal number threads per block
+   *
+   * https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#compute-capabilities
+   */
+  struct MaxThreadsPerBlock
+  {
+	// valid for sm_2.X - sm_7.5
+    BOOST_STATIC_ASSERT uint32_t value = 1024;
```

Comment:
I am getting a compile error here. Shouldn't there be parentheses around any static assert? Or what does BOOST_STATIC_ASSERT do here?

Reply:
ohh it is a copy past mistake it must be `BOOST_STATIC_CONSTEXPR`